api/router.go

```diff
@@ -25,25 +29,25 @@ func buildPrimaryRouter() *httprouter.Router {
 func methodNotAllowedFn(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(http.StatusMethodNotAllowed)
-	b, err := json.Marshal(_responses.MethodNotAllowed())
+	reponse, err := json.Marshal(_responses.MethodNotAllowed())
```

Comment:
Is this hungarian notation? ^^

Regarding readability, I think a longer, more explicit variable name makes it easier to read. Compare (assuming you are new to the codebase):
```
+ w.Write(response)
- w.Write(b)
```
I think one is way easier to read, than the other. As with `ds` above, I would argue that short variable names are not something to strive for, just "because".

Reply:
It's pragmatic naming more than anything: `response` generally denotes a struct or object which represents what is about to be returned, but here we're serializing it, so we shorthand it to `b`. 

For `ds` later on, the feature is called "datastores", so we ensure we're using the same name, but `datastore` is too close to a package name and can add confusion, thus `ds`.